typescript/api-client/projects/vcd/schematics/src/schematics/plugin-seed/index.ts

```diff
@@ -115,7 +115,7 @@ function addNewPlugin(folderName: string, pluginConfig: SourceFile) {
   return new InsertChange(
     `${PLUGINS_CONTAINING_FOLDER}/index.ts`,
     lastPluginItem,
-    `\tnew PluginRegistration("src/plugins/${folderName}/src", "main/subnav-plugin.module#SubnavPluginModule", "Seed Plugin"),\n`)
+    `\tnew PluginRegistration("src/plugins/${options.name}/src", "main/${strings.dasherize(options.module)}.module#${options.module}Module", "${options.module}"),\n`)
```

Comment:
- src/plugins/${options.name} should be dasherized?
- "${options.module}" as the label in the menu should be the un-dasherized name?

If the name is Seed Plugin, the filepath should be `src/plugins/seed-plugin` and the Menu label should be "Seed Plugin"